docs/next/components/Search.tsx

```diff
@@ -71,7 +71,7 @@ export function Search() {
         <link
           rel="preconnect"
           href={`https://${process.env.NEXT_PUBLIC_ALGOLIA_APP_ID_TEST}-dsn.algolia.net`}
-          crossOrigin="true"
+          crossOrigin="use-credentials"
```

Comment:
Why?

Reply:
I think either just `crossOrigin` or `crossOrigin="anonymous"` would be fine. (It actually looks like our React types are out of date in docs/next, so I'll take care of that and clean this up while I'm at it.)

Reply:
Oh I see, the type insists on a string, and without an attribute value it's treated as a boolean. Right. In that case, `crossOrigin=""` looks like it should be fine. (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin)

I'll go ahead and do the React types update anyway. :)

Reply:
dug into it. seems like `crossOrigin="true"` was never valid and it just falls back to "anonymous". anonymous it is then.

Reply:
ok - nvm saw you updated it in https://github.com/dagster-io/dagster/pull/19190